.env.template

```diff
@@ -0,0 +1,13 @@
+ZULIP_API_KEY=<your-zulip-api-key>
+SECRET_KEY=<your-secret-key>
+EMAIL_USER=<email-address>
+EMAIL_PASS=<your-Password>
+CONFIRM_PASS=<your-Password>
+
+SENDGRID_API_KEY=<your-sendgrid-api-key>
+DB_BACKEND=postgresql
+DB_NAME=<db-name>
+DB_USERNAME=<db-user-username>
+DB_PASSWORD=<db-user-password>
+DB_HOST=localhost
+DB_PORT=5432
```

Comment:
add extra line here

Reply:
Sure

Reply:
@Amulya-coder Please add the extra line at the end of the file.
It is showing no newline at the end of the file.